app/gui/src/project-view/components/GraphEditor/GraphNodeMessage.vue

```diff
@@ -84,4 +126,7 @@ export const colorForMessageType: Record<MessageType, string> = {
 .copyButton:active {
   color: var(--color-text-inversed);
 }
+.quickFixButton:active {
+  color: var(--color-text-inversed);
+}
```

Comment:
I think these two could be merged with`& > .SvgButton:active` above; I don't know why the copyButton was separate.

Reply:
OK, reusing the style of copy button in dd6e07cb0c

Reply:
Integrated as 6e06f3c145